### PR TITLE
Ethics page

### PR DESCRIPTION
**Pull request by korbinib**
included reviewer comments and new tools

### CODE REVIEW

File: pages/your_tasks/ethics.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    * Balancing access to research data and privacy of research participants (see [How much data can be collected](https://commission.europa.eu/law/law-topic/data-protection/reform/rules-business-and-organisations/principles-gdpr/how-much-data-can-be-collected_en), the [EU clinical trials regulation](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=celex%3A32014R0536) and respective national health research legislations)
+    * Unintentional discrimination of research participants
+    * Findings about participants that are outside of the original research question(s): see [Purpose of data processing](https://commission.europa.eu/law/law-topic/data-protection/reform/rules-business-and-organisations/principles-gdpr/purpose-data-processing_en) and [Incidental findings Archives - BBMRI-ERIC](https://www.bbmri-eric.eu/elsi-topic/incidental-findings/)
+​
+* Other ethical questions are arising from the impact of research outcomes, including data on the interest of communities or individuals
+    * Fair management of intellectual property rights - also see e.g. ABS
```
[the remark this answers is not in the document]
Comment (korbinib) [reply pages/your_tasks/ethics.md:47]:
```suggestion
    * Fair management of intellectual property rights - also see e.g. Access and Benefit-sharing (ABS)
```

File: pages/your_tasks/ethics.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    * Publication of research data that might impact the reputation of communities or individuals
+    * Publication of research data that might impact economical interests of  communities or individuals
+    * Publication of research data that might impact security of society, communities or individuals
```
[the remark this answers is not in the document]
Comment (korbinib) [reply pages/your_tasks/ethics.md:50]: I think there a different considerations for both, so I would suggest to keep them seperate

File: pages/your_tasks/ethics.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    * How can harm to other beings and the environment be identified and mitigated in a timely manner?
+​
+### Solutions
+* Assess potential ethical implication through an ethics review
+    * Your local ethics committee can help you to review the ethical implications of the project or might guide you to more relevant bodies and resources
+* In order to address challenges when working with human data (see also the [RDMkit page on human data](human_data))
+    * Used standardised consent forms (see e.g. [GA4GH consent toolkit]({% raw %}https://www.ga4gh.org/resource-library/#{%22document%22:{%22category%22:%22Toolkit%20Component%22,%22related_products%22:%22Consent%20Toolkit%22}}{% endraw %}), [BBMRI](https://zenodo.org/record/3928446)) that can be represented in a machine actionable way ([DUO](https://www.ga4gh.org/product/data-use-ontology-duo/))
+    * Anticipate the possibility of consent/data withdrawal and implement administrative and technical processes before you start collecting/processing, be transparent about these in the consent form and also about the cases when withdrawal is no longer possible due to anonymization, aggregation, or other data processing.
```
[the remark this answers is not in the document]
Comment (korbinib) [reply pages/your_tasks/ethics.md:68]:
```suggestion
    * Before you start collecting/processing data, be transparent about these in the consent form and also about the cases when withdrawal is no longer possible due to anonymization, aggregation, or other data processing.  Anticipate the possibility of consent/data withdrawal and implement administrative and technical processes.
```

File: pages/your_tasks/ethics.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    * Free and informed consent, the possibility of its withdrawal, its documentation, withdrawal (connected with data deletion) and its representation in a machine-actionable way (see also [What information must be given individuals whose data is collected](https://commission.europa.eu/law/law-topic/data-protection/reform/rules-business-and-organisations/principles-gdpr/what-information-must-be-given-individuals-whose-data-collected_en), the [EU clinical trials regulation](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=celex%3A32014R0536) and respective national health research legislations)
+    * Balancing access to research data and privacy of research participants (see [How much data can be collected](https://commission.europa.eu/law/law-topic/data-protection/reform/rules-business-and-organisations/principles-gdpr/how-much-data-can-be-collected_en), the [EU clinical trials regulation](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=celex%3A32014R0536) and respective national health research legislations)
+    * Unintentional discrimination of research participants
```
[the remark this answers is not in the document]
Comment (korbinib) [reply pages/your_tasks/ethics.md:43]: @Nazeeefa does this work as an example?

File: pages/your_tasks/ethics.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    * Findings about participants that are outside of the original research question(s): see [Purpose of data processing](https://commission.europa.eu/law/law-topic/data-protection/reform/rules-business-and-organisations/principles-gdpr/purpose-data-processing_en) and [Incidental findings Archives - BBMRI-ERIC](https://www.bbmri-eric.eu/elsi-topic/incidental-findings/)
+​
+* Other ethical questions are arising from the impact of research outcomes, including data on the interest of communities or individuals
+    * Fair management of intellectual property rights - also see e.g. ABS
+    * Publication of research data that might impact the reputation of communities or individuals
+    * Publication of research data that might impact economical interests of  communities or individuals
+    * Publication of research data that might impact security of society, communities or individuals
+​
+* General research ethics considerations that are also reflecting on research data, including:
```
[the remark this answers is not in the document]
Comment (korbinib) [reply pages/your_tasks/ethics.md:52]:
```suggestion
* There are also general research ethics considerations that are relevant in the context of research data, including:
```

File: pages/your_tasks/ethics.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Nazeeefa) [reply pages/your_tasks/ethics.md:43]: 'such as migrants' part is not necessary.

File: pages/your_tasks/ethics.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+    * Create processes for incidental findings before you start collecting data, include the way the participant wants you to deal with it in the consent form
+* In order to manage the impact of data collection and sharing:
+    * The management of intellectual property rights connected with the data, if any, should be planned early on ([RDMkit data management plan page](dmp)), be part of collaboration/consortium agreements, and make use of [standard licensing terms](licensing)
+    * The Nagoya Protocol regulates access to genetic resources and conditions for transfer of genetic resources and traditional knowledge across counties. For implementations into national law please consult the [ABS Clearing House](https://absch.cbd.int/en/) (also see [RDMkit compliance monitoring & measurement](compliance))  
+    * Laws & Regulations concerning biosecurity, data export control and national interests might be linked from the [RDMkit national resources pages](national_resources)
+    * In order to ensure conformity with ethical research principles, the following should also be considered for data management:
+        * Follow general research ethics laws and guidelines (see below)
```
Comment (Nazeeefa) [inline pages/your_tasks/ethics.md:77]: Is it possible to link sub-section where it says 'see below'?

Comment (korbinib) [reply pages/your_tasks/ethics.md:77]:
```suggestion
        * Follow general research ethics laws and guidelines (see [below](ethics#how-can-i-identify-regulations-guidelines-and-laws-connected-to-ethics-in-my-research-context))
```

Comment (korbinib) [reply pages/your_tasks/ethics.md]: @Nazeeefa added link

File: pages/your_tasks/ethics.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+        * Automated sharing of research data after a specific period or milestones and deliverables in a project can be a good mechanism to enhance the openness of a project ([data management coordination](dm_coordination))
+​
+
+## How can I identify regulations, guidelines and laws connected to ethics in my research context?
+​
+### Considerations
+* In all cases, your institution’s Data Protection Officer (DPO) is the person to refer to when considering ethical and legal aspects of data management.
+* When looking for recommendations and regulations, it is best to start from the local, that is, starting what is applicable within your discipline, and nationally. Then (if applicable), EU policies, directives and regulations are to be explored, as well as global recommendations (for example, from the UNESCO).
+* The list of resources given below does not attempt to be exhaustive: it tries to highlight the most relevant ones.
```
Comment (Nazeeefa) [inline pages/your_tasks/ethics.md:89]: Which list does this refer to? :) maybe you can rewrite this to make it clear, thanks

File: pages/your_tasks/ethics.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+    * For animal research: the [Three Rs principle](https://ec.europa.eu/health/scientific_committees/opinions_layman/en/non-human-primates/glossary/tuv/three-rs-principle.htm)  
+    * For data produced with Machine Learning (ML) techniques: the [recommendations from UNESCO](https://www.unesco.org/en/artificial-intelligence/recommendation-ethics), and the [DOME guidelines](https://dome-ml.org/), a product of the [ELIXIR ML focus group](https://elixir-europe.org/focus-groups/machine-learning)  
+* Resources and Policies from research infrastructures
+    * {% tool "bbmri-eric-s-elsi-knowledge-base" %} and the BBMRI [Helpdesk and Services](https://www.bbmri-eric.eu/elsi/helpdesk/)
+    * ELIXIR ELSI Policy [ELIXIR Ethics Policy](https://zenodo.org/record/557027#.ZGtrzpFBxD8)  
+* Check existing EU policies, directives and regulations
+    * The [General Data Protection Regulation](https://gdprinfo.eu/)
+    * The European [Clinical Trials Regulation](https://www.ema.europa.eu/en/human-regulatory/research-development/clinical-trials/clinical-trials-regulation)
+
+### Further materials
```
Comment (Nazeeefa) [inline pages/your_tasks/ethics.md:110]: do you plan to add links to these?

File: pages/your_tasks/ethics.md (hunk 7 above)
Comment (korbinib) [reply pages/your_tasks/ethics.md:89]:
```suggestion
* The solutions given below do not attempt to be exhaustive and highlight only the most relevant ones.
```

File: pages/your_tasks/ethics.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+title: Ethical aspects
+contributors: [Karel Berka , Erin Calhoun, Paulette Lieby, Anamika Chatterjee, Korbinian Bösl]
+description: Working on aspects in the management of research data that can raise ethical issues
+page_id: ethics
+related_pages:
+  tool_assembly: [covid-19, transmed, tsd, csc]
+training:
+- name: 'Ethics/ELSI considerations - From FAIR to fair data sharing - recorded webinar'
+  registry: TeSS
+  url: https://tess.elixir-europe.org/materials/ethics-elsi-considerations-from-fair-to-fair-data-sharing
```
[the remark this answers is not in the document]
Comment (korbinib) [reply pages/your_tasks/ethics.md]: changed in 007a321a4416ca993637150cf8da67240f331b6d

File: pages/your_tasks/ethics.md (hunk 8 above)
Comment (korbinib) [reply pages/your_tasks/ethics.md:104]: changed in 007a321a4416ca993637150cf8da67240f331b6d

File: pages/your_tasks/ethics.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+    * Prevention of withholding of research data
+    * Prevention of manipulation and fraud of research data
+    * Assessment of who is excluded or included to data access and why
+    * How can harm to other beings and the environment be identified and mitigated in a timely manner?
+​
+### Solutions
+* Assess potential ethical implication through an ethics review
+    * Your local ethics committee can help you to review the ethical implications of the project or might guide you to more relevant bodies and resources
+* In order to address challenges when working with human data (see also the [RDMkit page on human data](human_data))
+    * Used standardised consent forms (see e.g. [GA4GH consent toolkit]({% raw %}https://www.ga4gh.org/resource-library/#{%22document%22:{%22category%22:%22Toolkit%20Component%22,%22related_products%22:%22Consent%20Toolkit%22}}{% endraw %}), [BBMRI](https://zenodo.org/record/3928446)) that can be represented in a machine actionable way ([DUO](https://www.ga4gh.org/product/data-use-ontology-duo/))
```
Comment (Nazeeefa) [inline pages/your_tasks/ethics.md:61]: This point needs to revised, including the link to consent toolkit - it's not displayed properly in preview. Also, there needs to be some more/clear information in order to mention DUO and the GA4GH link for it.

Comment (korbinib) [reply pages/your_tasks/ethics.md:61]:
```suggestion
    * Used standardised consent forms (see e.g. 
{% tool "ga4gh-regulatory-and-ethics-toolkit" %}) with clauses that can be represented in a machine actionable way using the {% tool "data-use-ontology" %} and {% tool "informed-consent-ontology" %})
```

File: pages/your_tasks/ethics.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+
+### Further materials
+* Basic ethical principles are compliant with relevant international regulations:
+    * [Universal Declaration of Human Rights, 10 December 1948](https://www.un.org/en/about-us/universal-declaration-of-human-rights)
+    * [Article 8 of the European Convention on Human Rights](https://rm.coe.int/CoERMPublicCommonSearchServices/DisplayDCTMContent?documentId=090000168007ff47)
+    * [Council of Europe Convention for the Protection of Individuals with Regard to Automatic Processing of Personal Data 1981 (convention 108)](https://www.coe.int/en/web/conventions/full-list?module=treaty-detail&treatynum=108)
+    * [Charter of Fundamental Rights of the European Union 2010/C 83/0215](https://eur-lex.europa.eu/LexUriServ/LexUriServ.do?uri=OJ:C:2010:083:0389:0403:en:PDF)
+    * [Directive 95/46/EC of the European Parliament of 24 October 1995 on the protection of individuals with regard to the processing of Personal Data and on the free movement of such data](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=celex%3A31995L0046)https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=celex%3A31995L0046
```
Comment (korbinib) [inline pages/your_tasks/ethics.md:110]:
```suggestion
    * [Directive 95/46/EC of the European Parliament of 24 October 1995 on the protection of individuals with regard to the processing of Personal Data and on the free movement of such data](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=celex%3A31995L0046)
```

**Review by Nazeeefa — APPROVED** [review]

File: pages/your_tasks/ethics.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+    * [WMA Declaration of Helsinki – Ethical Principles for Medical Research Involving Human Subjects](https://www.wma.net/policies-post/wma-declaration-of-helsinki-ethical-principles-for-medical-research-involving-human-subjects/)  
+    * For animal research: the [Three Rs principle](https://ec.europa.eu/health/scientific_committees/opinions_layman/en/non-human-primates/glossary/tuv/three-rs-principle.htm)  
+    * For data produced with Machine Learning (ML) techniques: the [recommendations from UNESCO](https://www.unesco.org/en/artificial-intelligence/recommendation-ethics), and the [DOME guidelines](https://dome-ml.org/), a product of the [ELIXIR ML focus group](https://elixir-europe.org/focus-groups/machine-learning)  
+* Resources and Policies from research infrastructures
+    * {% tool "bbmri-eric-s-elsi-knowledge-base" %} and the BBMRI [Helpdesk and Services](https://www.bbmri-eric.eu/elsi/helpdesk/)
+    * ELIXIR ELSI Policy [ELIXIR Ethics Policy](https://zenodo.org/record/557027#.ZGtrzpFBxD8)  
+* Check existing EU policies, directives and regulations
+    * The [General Data Protection Regulation](https://gdprinfo.eu/)
```
Comment (Nazeeefa) [inline pages/your_tasks/ethics.md:101]: might be worth pointing to https://rdmkit.elixir-europe.org/gdpr_compliance here or earlier in text. Just a suggestion

**Pull request merged**